src/app/query/bitcoin/ordinals/brc20/brc20-tokens.query.ts

```diff
@@ -53,15 +61,65 @@ async function fetchBrc20TokensByAddress(address: string): Promise<Brc20Token[]>
   });
 }
 
-type FetchBrc20TokensByAddressResp = Awaited<ReturnType<typeof fetchBrc20TokensByAddress>>;
+export function useBrc20TokensQuery() {
+  const network = useCurrentNetwork();
+  const nativeSegwitSigner = useCurrentAccountNativeSegwitIndexZeroSigner();
+  const currentBitcoinAddress = nativeSegwitSigner.address;
+  const createSigner = useCurrentAccountTaprootSigner();
+
+  const getTaprootAddressData = useCallback(
```

Comment:
```suggestion
  const getNextTaprootAddressBatch = useCallback(
```

Suggestion for more readable name?

or 
<img width="447" alt="image" src="https://github.com/hirosystems/wallet/assets/1618764/3612a5b9-7856-47d7-809d-68a092ba2c4d">

To me "data" implies that there's some metadata beyond the addresses themselves